xplique/commons/operators.py

```diff
@@ -7,7 +7,7 @@
 import tensorflow as tf
 
 from ..types import Callable, Optional
-from ..utils_functions.object_detection import _box_iou, _format_objects, _EPSILON
```

Comment:
I agree that the epsilon should not be loaded simultaneously as loading operators. But is it a good thing to initialize it at each run of the tf function?